hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/RegionServerListTmpl.jamon

```diff
@@ -97,9 +97,13 @@ Arrays.sort(serverNames);
     int totalRegions = 0;
     int totalRequestsPerSecond = 0;
     int inconsistentNodeNum = 0;
+    String state = "Normal";
     String masterVersion = VersionInfo.getVersion();
+    List<ServerName> decommissionedServers = master.listDecommissionedRegionServers();
```

Comment:
Better construct a Set here? The contains method run on List is O(n), although usually the number of decommissioned server will be small, but it is still no harm to use a Set here.

Reply:
Thanks @Apache9 for your comment. I will update it later.